myapp/views.py

```diff
@@ -0,0 +1,87 @@
+from flask import render_template, request, redirect, flash,url_for
+from main import app
+
+#This file contains all the views
+
+@app.route('/')
+def index():
+    return render_template("layout.html")
+
+
+# @app.route('/gifs/<gif>')
```

Comment:
I doubt we'll want either hello or form views in our final project, so it's probably best to remove them.